test/integration/analytics_int_test.go

```diff
@@ -90,7 +102,6 @@ func (suite *AnalyticsIntegrationTestSuite) TestActivateEvents() {
 			cp.Snapshot(), ts.MostRecentStateLog(), ts.SvcLog()))
 
 	cp.SendLine("exit")
-	cp.ExpectExitCode(0)
```

Comment:
Please file a followup ticket for us to investigate this. We shouldn't drop assertions because they are problematic without understanding why they are problematic.

Reply:
Filed here: https://activestatef.atlassian.net/browse/DX-994